stable/database/README.md

```diff
@@ -331,6 +331,17 @@ The following tables list the configurable parameters of the `database` chart an
 | `te.readinessProbe.failureThreshold` | The number of times that the readiness probe must fail before the container is marked as unready. | `3` |
 | `te.readinessProbe.successThreshold` | The number of times that the readiness probe must success before the container is marked as ready. | `1` |
 | `te.readinessProbe.timeoutSeconds` | The timeout in seconds for an invocation of the readiness probe. | `5` |
+| `te.autoscaling.enabled` | Whether to enable auto-scaling for TE deployment. | `false` |
+| `te.autoscaling.minReplicas` | The lower limit for the number of TE replicas to which the autoscaler can scale down. | `1` |
+| `te.autoscaling.maxReplicas` | The upper limit for the number of TE replicas to which the autoscaler can scale up. It cannot be less than the minReplicas. | `3` |
+| `te.autoscaling.hpa.targetCpuUtilization` | The target average CPU utilization value across all TE pods, represented as a percentage. | `80` |
+| `te.autoscaling.hpa.behavior` | Configures the scaling behavior of the target in both Up and Down directions. | `...` |
+| `te.autoscaling.hpa.behavior.scaleUp.stabilizationWindowSeconds` | The number of seconds for which past recommendations should be considered while scaling up. | `300` |
+| `te.autoscaling.keda.enabled` | Whether to enable auto-scaling for TE deployment by using KEDA ScaledObject resource instead of HPA. | `false` |
+| `te.autoscaling.keda.pollingInterval` | This is the interval in seconds to check each trigger on. | `30` |
+| `te.autoscaling.keda.cooldownPeriod` | The period in seconds to wait after the last trigger reported active before scaling the resource back to 0. | `300` |
+| `te.autoscaling.keda.fallback` | The fallback section is optional. It defines a number of replicas to fall back to if a scaler is in an error state. | `{}` |
```

Comment:
Nit: Is "The fallback section is optional" necessary? I would assume any value that has a default is optional.

Reply:
I also can't tell from this what the format or subproperties are. Can you link the documentation for KEDA if this is just exposing a KEDA configuration verbatim?